website/content/en/docs/upgrading-sdk-version/v1.0.0-alpha.1.md

```diff
@@ -0,0 +1,209 @@
+
+_See [#3466](https://github.com/operator-framework/operator-sdk/pull/3466) and [#3451](https://github.com/operator-framework/operator-sdk/pull/3451) for more details._
+
+## Removed `pkg/k8sutil`
+
+With the transition to Kubebuilder-style projects, `pkg/k8sutil` is no longer used in the default scaffolding
+for Go operators. Migrate your project to the new Kubebuilder-style layout to remove the need for this
+package.
+
+_See [#3475](https://github.com/operator-framework/operator-sdk/pull/3475) for more details._
+
+## Removed `pkg/kube-metrics` and `pkg/metrics`
+
+Remove the call to `addMetrics` in your `main.go` file and begin using the
+`InstrumentedEnqueueRequestForObject` handler when setting up controller-runtime
+watches for your primary CRs.
+
+`InstrumentedEnqueueRequestForObject` can be imported from `github.com/operator-framework/operator-lib/handler`
+
+_See [#3484](https://github.com/operator-framework/operator-sdk/pull/3484) for more details._
+
+## Removed package `pkg/ready`
+
+Use `controller-runtime`'s readyz server that supports custom http handlers. Add add a `healthz.Checker` (e.g. [`healthz.Ping`](
+https://github.com/kubernetes-sigs/controller-runtime/blob/229c3c357d9e6b07c3d6774010c35161a82b08f9/pkg/healthz/healthz.go#L187-L188)) 
+using [`manager.AddReadyzCheck`](https://github.com/kubernetes-sigs/controller-runtime/blob/229c3c357d9e6b07c3d6774010c35161a82b08f9/pkg/manager/manager.go#L73-L74)
+
+_See [#3476](https://github.com/operator-framework/operator-sdk/pull/3476) for more details._
+
+## Removed package `pkg/tls`
+
+See the [Kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/cert-manager.html#deploying-the-cert-manager) on
+how to deploy and manage TLS certificates with cert-manager.
+
+_See [#3468](https://github.com/operator-framework/operator-sdk/pull/3468) for more details._
+
+## Update your scorecard config file to the new format
+
+See the updated scorecard [config documentation](https://sdk.operatorframework.io/docs/scorecard/scorecard/#config-file)
+for details.
+
+_See [#3434](https://github.com/operator-framework/operator-sdk/pull/3434) and
+[#3490](https://github.com/operator-framework/operator-sdk/pull/3490) for more details._
+
+## Move scorecard/alpha to scorecard
+
+If you have been using `operator-sdk alpha scorecard`, update to use `operator-sdk scorecard`.
+<!-- TODO(joelanford): update this link to sdk.operatorframework.io when 1.0.0 is released -->
+If you have been using `operator-sdk scorecard`, migrate to the new scorecard. See the new
+[scorecard documentation](https://master.sdk.operatorframework.io/docs/advanced-topics/scorecard/scorecard/)
+
+_See [#3444](https://github.com/operator-framework/operator-sdk/pull/3444) for more details._
+
+## Scorecard output formatting has changed
+
+Update any scripts interpretting the scorecard output to
+understand the v1alpha3 TestList format.
```

Comment:
Nit: I realize this is coming straight from the fragment but we should probably format this the same as it is in the CHANGELOG. Esp since it's a struct/var name.
```suggestion
understand the `v1alpha3.TestList` format.
```